base/applications/cmdutils/wscript/host.c

```diff
@@ -407,7 +407,14 @@ static HRESULT WINAPI Host_DisconnectObject(IHost *iface, IDispatch *Object)
 
 static HRESULT WINAPI Host_Sleep(IHost *iface, LONG Time)
 {
+#ifdef __REACTOS__
```

Comment:
```suggestion
#ifdef __REACTOS__
    UNREFERENCED_PARAMETER(iface);
```
Or what should this parameter be used for?

Reply:
> Or what should this parameter be used for?

This is just how the COM ABI is in C. In C++ it would be the hidden `this`.